packages/react-avatar/src/components/Avatar/Avatar.test.tsx

```diff
@@ -2,11 +2,42 @@ import * as React from 'react';
 import { isConformant } from '../../common/isConformant';
 import { Avatar } from './Avatar';
 import { render, screen } from '@testing-library/react';
+import { avatarClassNames } from './useAvatarStyles';
 
 describe('Avatar', () => {
   isConformant({
     Component: Avatar,
     displayName: 'Avatar',
+    testOptions: {
+      'has-static-classnames': [
+        {
+          props: {
+            image: { src: 'avatar.png', alt: 'test-image' },
+            initials: 'Test Initials',
+            badge: 'Test Badge',
+          },
+          expectedClassNames: {
+            root: avatarClassNames.root,
+            image: avatarClassNames.image,
+            initials: avatarClassNames.initials,
+            badge: avatarClassNames.badge,
+          },
```

Comment:
```suggestion
          expectedClassNames: {...avatarClassNames}
```

this would work right?

Reply:
It won't in `Avatar` as the `initials` and `icon` slots are mutually exclusive -- when `icon` is rendered `initials` will not be rendered. This is why `testOptions` takes an array of "variants". I'm not in love with this solution but I know there are some plans to rework conformance tests so I feel like this approach is "good enough" for now and we can roll this need into a refactor. Currently `Avatar` is the only component with this mutually exclusive case but I imagine future components with greater complexity will have this behavior as well.